src/Core/OAuth/OAuth2/OAuth2LoginHelper.php

```diff
@@ -228,11 +228,12 @@ public function getAuthorizationCodeURL(){
      *
      * @param String $new_log_location     The directory path for storing request and response log
      *
+     * @return $this
      */
     public function setLogForOAuthCalls($enableLogs, $debugMode, $new_log_location = null)
     {
         if ($enableLogs) {
-          $this->RequestLogging = new LogRequestsToDisk($enableLogs, $new_log_location);
+          $this->RequestLogging = new LogRequestsToDisk(true, $new_log_location);
```

Comment:
@abisalehalliprasan Based on $enableLogs value at ln 235,it is going inside the function. If it true, why do we need to change at Ln236.